### PR TITLE
Add the System Alerts block to the Customer Engagement Reporting page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- adds the System Alerts block to the Customer Engagement Reporting page
- adds new step to the Import Settings block

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/reports/customer-engagement.html

### CODE REVIEW

File: src/reports/customer-engagement.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ To directly link your Engagement Cloud and Magento accounts without requiring a
    1. Set the **Contact Sync Size** (default 500). This sets the batch size limit you require for the amount of contacts loaded for every sync.
    1. Set the **Transactional Data Sync Size** (default 50). This sets the batch size limit you require for the amount of transactions loaded for every sync.
    1. To sync subscriber dales data, set **Enable Subscribers Sales Data Sync** to "Yes". This option is disabled by default due to the increased server load and sync times.
+   1. To remove the /pub directory from image URLs in the catalog, set Remove /pub directory from URLs to “Yes”.
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:66]:
```suggestion
   1. To remove the /pub directory from image URLs in the catalog, set **Remove /pub directory from URLs** to `Yes`.
```

File: src/reports/customer-engagement.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ To directly link your Engagement Cloud and Magento accounts without requiring a
    1. Set the **Contact Sync Size** (default 500). This sets the batch size limit you require for the amount of contacts loaded for every sync.
    1. Set the **Transactional Data Sync Size** (default 50). This sets the batch size limit you require for the amount of transactions loaded for every sync.
    1. To sync subscriber dales data, set **Enable Subscribers Sales Data Sync** to "Yes". This option is disabled by default due to the increased server load and sync times.
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:65]: Let's fix this one too
```suggestion
   1. To sync subscriber dales data, set **Enable Subscribers Sales Data Sync** to `Yes`. (This option is disabled by default due to the increased server load and sync times.)
```

File: src/reports/customer-engagement.md [hunk 3]
```diff
@@ -149,4 +150,14 @@ If you leave this field empty, a security check does not run. We do not recommen
 1. Click <span class="btn">Save Config</span>.
 
     ![Debug]({% link images/images/dotdigital-dash-iprestrict.png %}){: .zoom}
-    _Debug_
\ No newline at end of file
+    _Debug_
+
+## Configure System Alerts
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:155]:
```suggestion
## Configure system alerts
```

File: src/reports/customer-engagement.md [hunk 4]
```diff
@@ -149,4 +150,14 @@ If you leave this field empty, a security check does not run. We do not recommen
...
+
+1. To send in-app system messages to admin users when errors that may affect the dotdigital extension are detected, set **Enable System Messages** to "Yes".
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:157]:
```suggestion
1. To send system messages to Admin users when errors that could affect the dotdigital extension are detected, set **Enable System Messages** to `Yes`.
```

File: src/reports/customer-engagement.md [hunk 5]
```diff
@@ -149,4 +150,14 @@ If you leave this field empty, a security check does not run. We do not recommen
...
+1. To send system alert email notifications when errors that may affect the dotdigital extension are detected, set **Enable Email Notifications** to "Yes".
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:158]:
```suggestion
1. To send system alert email notifications when errors that could affect the dotdigital extension are detected, set **Enable Email Notifications** to `Yes`.
```

File: src/reports/customer-engagement.md [hunk 6]
```diff
@@ -149,4 +150,14 @@ If you leave this field empty, a security check does not run. We do not recommen
...
+1. Select the user role in the **Select Roles** field  who will receive email notifications.
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:159]:
```suggestion
1. For **Select Roles**, select the user role to determine who will receive email notifications.
```

File: src/reports/customer-engagement.md [hunk 7]
```diff
@@ -149,4 +150,14 @@ If you leave this field empty, a security check does not run. We do not recommen
...
+1. To specify the email notification frequency and the time interval for error reporting, set **Alert Frequency** to one of the options: 1 hour, 12 hours, 24 hours, 48 hours, 72 hours.
```
Comment (leslietilling) [inline src/reports/customer-engagement.md:160]:
```suggestion
1. To specify the email notification frequency and the time interval for error reporting, set **Alert Frequency** to one of the following options:
   * 1 hour
   * 12 hours
   * 24 hours
   * 48 hours
   * 72 hours
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits

**Comment by ZhovnirMaria** [comment]
@leslietilling hi! 👋 
Thank you, done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Comment by dobooth** [comment]
Auto-close failed due to merge conflict.

**Pull request closed without merging**